.yarn/versions/92db221e.yml

```diff
@@ -0,0 +1,5 @@
+releases:
+  "@yarnpkg/plugin-compat": patch
+
+declined:
+  - "@yarnpkg/cli"
```

Comment:
`@yarnpkg/plugin-compat`'s list gets bundled inside the CLI so `@yarnpkg/cli` needs a patch release too.

Reply:
Fixed.